pkg/server/service/service.go

```diff
@@ -280,7 +322,14 @@ func (m *Manager) getLoadBalancerServiceHandler(ctx context.Context, serviceName
 		return nil, err
 	}
 
-	lb := wrr.New(service.Sticky, service.HealthCheck != nil)
+	// here choose between WRR and HRW based on load balancer type
+	var lbHRW *hrw.Balancer
+	var lbWRR *wrr.Balancer
+	if info.LoadBalancer.Type == "hrw" {
+		lbHRW = hrw.New(service.HealthCheck != nil)
+	} else {
+		lbWRR = wrr.New(service.Sticky, service.HealthCheck != nil)
+	}
```

Comment:
Wdyt about using `interface`, in order keep only one lb instead of multiple "if lb == hrw" ?